core/src/main/java/org/apache/carbondata/core/statusmanager/LoadMetadataDetails.java

```diff
@@ -460,4 +460,9 @@ public String getPath() {
   public void setPath(String path) {
     this.path = path;
   }
+
+  public boolean isCarbonSegment() {
```

Comment:
Better to change to `isCarbonFormat`. Since after this PR, CarbonData support mixed format, so all segments are carbon segments, but with different format

Reply:
ok